assets/xml/objects/object_lightblock.xml

```diff
@@ -1,9 +1,10 @@
 ﻿<Root>
+    <!-- Object for Lightblock -->
     <File Name="object_lightblock" Segment="6">
-        <DList Name="object_lightblock_DL_000170" Offset="0x170" /> <!-- Original name is "z2_brick_light_modelT" -->
-        <DList Name="object_lightblock_DL_000178" Offset="0x178" /> <!-- Original name is "z2_brick_light_model" -->
-        <Texture Name="object_lightblock_TLUT_000258" OutName="tlut_000258" Format="rgba16" Width="4" Height="4" Offset="0x258" />
-        <Texture Name="object_lightblock_Tex_000278" OutName="tex_000278" Format="ci4" Width="64" Height="64" Offset="0x278" />
-        <Collision Name="object_lightblock_Colheader_000B80" Offset="0xB80" /> <!-- Original name is "z2_brick_light_bgdatainfo" -->
+        <DList Name="gLightblockEmptyDL" Offset="0x170" /> <!-- Original name is "z2_brick_light_modelT" -->
+        <DList Name="gLightblockDL" Offset="0x178" /> <!-- Original name is "z2_brick_light_model" -->
+        <Texture Name="gLightblockTLUT" OutName="lightblock_tlut" Format="rgba16" Width="4" Height="4" Offset="0x258" />
+        <Texture Name="gLightblockTex" OutName="lightblock_tex" Format="ci4" Width="64" Height="64" Offset="0x278" />
```

Comment:
```suggestion
        <Texture Name="gLightblockTex" OutName="lightblock" Format="ci4" Width="64" Height="64" Offset="0x278" />
```
We try to avoid `_tex` on the outname